src/services/public_http_server/handlers/relay_webhook/error.rs

```diff
@@ -40,10 +44,16 @@ pub enum RelayMessageClientError {
     AppDoesNotMatch,
 
     #[error("Decode message: {0}")]
-    DecodeMessage(#[from] base64::DecodeError),
+    DecodeMessage(base64::DecodeError),
 
     #[error("Could not parse message envelope: {0}")]
-    EnvelopeParseError(#[from] EnvelopeParseError),
+    EnvelopeParseError(EnvelopeParseError),
```

Comment:
Consider removing the Error suffix?

Reply:
Did in [a71de03](https://github.com/WalletConnect/notify-server/pull/413/commits/a71de034558050e8211446dd1ff289a68a7d3898)